.github/workflows/make-charts-index.yaml

```diff
@@ -0,0 +1,76 @@
+name: Make Charts Index
+
+on:
+  workflow_dispatch:
+    inputs:
+      repoName:
+        type: options
+        required: true
+        description: "Repository name?"
+        default: "stable"
+        options:
+          - "stable"
+          - "canary"
+
+  workflow_call:
+    inputs:
+      repoName:
+        type: string
+        required: true
+        description: "Repository name?"
+        default: "stable"
+
+jobs:
+  query:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v3
+
+      - name: Install Helm
+        uses: azure/setup-helm@v3
```

Comment:
@cjorge-graphops not clear what azure is doing here. is this a preexisting workflow that we are just calling instead of defining ourselves?

Reply:
ignore this - it makes sense!